python/OnnxUdf/example.py

```diff
@@ -0,0 +1,35 @@
+import openeo
+
+# This will be required to import onnxruntime in the UDF.
```

Comment:
Also document how this zip file was created? So that users know how to adapt if they have other constraints.

I think it makes sense to add a README.md file for that kind of stuff